plugins/woocommerce-admin/client/products/product-form.tsx

```diff
@@ -8,13 +8,13 @@ import {
 } from '@woocommerce/components';
 import { PartialProduct, Product } from '@woocommerce/data';
 import { Ref } from 'react';
+import { PluginArea } from '@wordpress/plugins';
 
 /**
  * Internal dependencies
  */
 import { ProductFormHeader } from './layout/product-form-header';
 import { ProductFormLayout } from './layout/product-form-layout';
-import { ProductDetailsSection } from './sections/product-details-section';
```

Comment:
The `product-details-section.tsx` file under `sections` can be deleted now as well right?

Reply:
Right! I had this in mind and then entirely forgot. Done in https://github.com/woocommerce/woocommerce/pull/36368/commits/b1a447d3edb6dcf384cd74fd15120210b944fd4e, and I retested. All seems well.